### PR TITLE
Add release announcement for 1.16.3

### PR DESCRIPTION
**Pull request by morphis**

### CODE REVIEW

File: announcements/1.16.3.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+### Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.16.3.
+
+This is an out-of-order release to pin deployments of the Anbox Cloud Appliance to a new APT repository signing key for Grafana.
```
Comment (ru-fu) [inline announcements/1.16.3.md:5]:
```suggestion
This is an out-of-order release to pin deployments of the Anbox Cloud Appliance to a new signing key for the Grafana APT repository.
```

File: announcements/1.16.3.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+Grafana Labs has published [a note](https://grafana.com/blog/2023/01/12/grafana-labs-update-regarding-circleci-security-updates/) where they announced the revocation of the current signing key of their APT package repositories. Please read the blog post for more details.
```
Comment (ru-fu) [inline announcements/1.16.3.md:7]:
```suggestion
Grafana Labs has published [a note](https://grafana.com/blog/2023/01/12/grafana-labs-update-regarding-circleci-security-updates/) where they announce the revocation of the current signing key of their APT package repositories. See the blog post for more details.
```

File: announcements/1.16.3.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+The Anbox Cloud Appliance installs Grafana if the user accepts its installation at initialization time and pins the APT signing key internally to detect changes in the repository signature and fail installation if the provided signature is not correct. Due to this all new installations and updates of existing ones will fail until the new signing key is installed. The 1.16.3 release of the Anbox Cloud Appliance pins the new signing key from Grafana Labs and will allow installation and updates for Grafana again.
```
Comment (ru-fu) [inline announcements/1.16.3.md:9]:
```suggestion
If the user chooses to add support for monitoring during the initialisation of the Anbox Cloud Appliance, the appliance installs Grafana and pins the APT signing key internally. This is needed to detect changes in the repository signature and to fail installation if the provided signature is not correct. Due to this, all new installations and updates of existing installations will fail until the new signing key is installed. The 1.16.3 release of the Anbox Cloud Appliance pins the new signing key from Grafana Labs, allowing installation and updates for Grafana again.
```

File: announcements/1.16.3.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+If you have deployed the regular Anbox Cloud and installed [the Grafana charm yourself](https://charmhub.io/grafana), you can update the [`install_keys` configuration item](https://charmhub.io/grafana/configure#install_keys) to provide the new GPG signing key to the installation.
```
Comment (ru-fu) [inline announcements/1.16.3.md:11]:
```suggestion
If you deployed the regular Anbox Cloud and installed [the Grafana charm yourself](https://charmhub.io/grafana), update the [`install_keys` configuration item](https://charmhub.io/grafana/configure#install_keys) to provide the new GPG signing key to the installation.
```

File: announcements/1.16.3.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+n/a
+
+#### Bugs
+
+* [LP #2002776](https://bugs.launchpad.net/charm-grafana/+bug/2002776): Grafana APT repository key was rotated and charm needs to adapt
+
+## Upgrade instructions
+
+See [Upgrade Anbox Cloud](https://anbox-cloud.io/docs/howto/update/upgrade-anbox) or [Upgrade the Anbox Cloud Appliance](https://anbox-cloud.io/docs/howto/update/upgrade-appliance) for instructions on how to update your Anbox Cloud deployment to the 1.16.2 release.
```
Comment (ru-fu) [inline announcements/1.16.3.md:25]:
```suggestion
See [Upgrade the Anbox Cloud Appliance](https://anbox-cloud.io/docs/howto/update/upgrade-appliance) for instructions on how to update your Anbox Cloud Appliance deployment to the 1.16.3 release.
```

File: announcements/1.16.3.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
+### Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.16.3.
```
Comment (ru-fu) [inline announcements/1.16.3.md:3]: Should we clarify that the update is only for the appliance (but that there is an action required for the full deployment)?

**Review by ru-fu — COMMENTED** [review]
Also need to add GPG to the wordlist.

File: announcements/1.16.3.md (hunk 6 above)
Comment (morphis) [reply announcements/1.16.3.md:3]: That there is an action required for the full deployment is not clear. We don't provide Grafana with it. So all we can do is give a hint at the end that if someone has Grafana deployed, they need to update the key.

Comment (ru-fu) [reply announcements/1.16.3.md:3]: How about:
```suggestion
The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.16.3. This release updates only the Anbox Cloud Appliance. However, if you run the regular Anbox Cloud and have Grafana deployed, an action is required (see below).
```

**Review by ru-fu — APPROVED** [review]
Still need to add GPG to the wordlist. ;)

File: release-notes.md [hunk 7]
```diff
@@ -3,6 +3,20 @@
 See [How to upgrade Anbox Cloud](https://discourse.ubuntu.com/t/upgrading-from-previous-versions/17750) or [How to upgrade the Anbox Cloud Appliance](https://discourse.ubuntu.com/t/upgrade-anbox-cloud-appliance/24186) for
 instructions on how to update your Anbox Cloud deployment.
 
+[Details=1.16.3]
+
+## 1.16.3 (January 13 2023)
```
Comment (ru-fu) [inline release-notes.md:8]:
```suggestion
## 1.16.3 (January 17 2023)
```

**Review by ru-fu — APPROVED** [review]

**Pull request merged**